examples/llm/src/models/mosaic_gpt.py

```diff
@@ -134,37 +134,66 @@ def _attn_bias(self, device, dtype):
     def forward(
             self,
             input_ids: torch.LongTensor,
-            key_padding_mask: Optional[torch.ByteTensor] = None,
-            past_key_values: Optional[List[Tuple[torch.FloatTensor]]] = None):
+            past_key_values: Optional[List[Tuple[torch.FloatTensor]]] = None,
+            attention_mask: Optional[torch.ByteTensor] = None,
+            return_dict: Optional[bool] = None,
+            output_attentions: Optional[bool] = None,
+            output_hidden_states: Optional[bool] = None,
+            use_cache: Optional[bool] = None):
+        return_dict = return_dict if return_dict is not None else self.config.return_dict
+        use_cache = use_cache if use_cache is not None else self.config.use_cache
+        if not return_dict:
+            raise NotImplementedError(
+                'return_dict False is not implemented yet for MosaicGPT')
+        if output_attentions:
+            raise NotImplementedError(
+                'output_attentions is not implemented yet for MosaicGPT')
+        if output_hidden_states:
+            raise NotImplementedError(
+                'output_hidden_states is not implemented yet for MosaicGPT')
```

Comment:
do we want to point to a doc where these args are explained?
this'll help explain why we have the args, but then error out...

Reply:
let me add an explanatory comment in the code at least